gitbook-docs/market/cmd-reference/README.md

```diff
@@ -1,3 +1,5 @@
-# CMD Reference
+# CLI Reference
 
-Scripts to interact with the proof market. These are located in `scripts` directory.
+`=nil;` prepared a [toolchain](https://github.com/NilFoundation/proof-market-toolchain)
+for interacting with the Proof Market.
```

Comment:
There's something in this phrase that I don't like. The first thing is about preparing. We don't prepare it, we develop it. Next, there could be a more clear call to action.

Use the scripts in the [Proof Market toolchain]() to interact with the Proof Market:

* [Make a new account]()
* [Push your statement as a circuit developer]
* [Order proofs for a zkApp]
* [Propose to sell proofs]
* [Upload and download proofs]